### PR TITLE
C2LC-600: Add the Weavly Logo

### PR DESCRIPTION
**Pull request by the-t-in-rtf**
See [C2LC-600](https://issues.fluidproject.org/browse/C2LC-600) for details.

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1116,6 +1130,8 @@ export class App extends React.Component<AppProps, AppState> {
     }
 
     render() {
+        const Logo = getThemeLogo(this.state.settings.theme);
+        // TODO: Use different logo depending on theme.  Confirm working in preview.
```
Comment (sbates-idrc) [inline src/App.js:1134]: I think this TODO can be removed now?

**Comment by sbates-idrc** [comment]
This PR does not add any focus styling for the logo yet. Please add.

**Comment by sbates-idrc** [comment]
With the change of class name on the element containing the heading/logo, from `App__app-heading` to `App__logo-container`, there are left over styles for the removed `App__app-heading` class. Please remove them.

**Comment by the-t-in-rtf** [comment]
> This PR does not add any focus styling for the logo yet. Please add.

In chats we discussed this, there were issues in Firefox and Safari related to using an outline colour and the browser default focus behaviour for anchors.  I converted to use the same approach we used with the privacy pull.

**Pull request merged**